<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(85, 170, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>700</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F0F5FA;
border-radius: 15px; 
 box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="highTemp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Highest Temp:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wind">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Wind:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lowTemp">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Lowest Temp:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="humiditylabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>110</y>
       <width>350</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>Humidity:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>700</width>
      <height>170</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#F0F5FA;
border-radius: 15px; 
 box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="monday">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Mon</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="tuesday">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Tue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="wedenesday">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Wed</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="thursday">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Thur</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="friday">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Fri</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="saturday">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Sat</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sunday">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Sun</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sunny">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>sunny.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="rainy">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>rainy.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="thunder">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>thunder.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="cloudy">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>cloudy.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="snowy">
     <property name="geometry">
      <rect>
       <x>425</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>snowy.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="foggy">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>foggy.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="humidity">
     <property name="geometry">
      <rect>
       <x>625</x>
       <y>50</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>humidity.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayone">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string> 03/29</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayonetemp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>35°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daythreetemp">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>27°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daytwotemp">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>28°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayseventemp">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>27°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayfivetemp">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>25°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayfourtemp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>29°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daysixtemp">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>27°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daytwo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string> 03/30</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daythree">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string> 03/31</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayfour">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>04/01</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayfive">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>04/02</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="dayseven">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string> 04/04</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="daysix">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>130</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 700 italic  12pt &quot;Segoe UI&quot;;
color: black;
</string>
     </property>
     <property name="text">
      <string>  04/03</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <zorder>monday</zorder>
    <zorder>tuesday</zorder>
    <zorder>wedenesday</zorder>
    <zorder>thursday</zorder>
    <zorder>friday</zorder>
    <zorder>saturday</zorder>
    <zorder>sunday</zorder>
    <zorder>rainy</zorder>
    <zorder>thunder</zorder>
    <zorder>cloudy</zorder>
    <zorder>snowy</zorder>
    <zorder>foggy</zorder>
    <zorder>humidity</zorder>
    <zorder>dayone</zorder>
    <zorder>dayonetemp</zorder>
    <zorder>daythreetemp</zorder>
    <zorder>daytwotemp</zorder>
    <zorder>dayseventemp</zorder>
    <zorder>dayfivetemp</zorder>
    <zorder>dayfourtemp</zorder>
    <zorder>daysixtemp</zorder>
    <zorder>daytwo</zorder>
    <zorder>daythree</zorder>
    <zorder>dayfour</zorder>
    <zorder>dayfive</zorder>
    <zorder>dayseven</zorder>
    <zorder>daysix</zorder>
    <zorder>sunny</zorder>
   </widget>
   <widget class="QLabel" name="city">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>761</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic  13pt &quot;Segoe UI&quot;;
color: black;
</string>
    </property>
    <property name="text">
     <string>   Bangkok</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="citysearchlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>152</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700  13pt &quot;Segoe UI&quot;;

color: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>  City</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>110</y>
      <width>456</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 51);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterbutton">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700  10pt &quot;Segoe UI&quot;;
background-color: #F0F5FA;
color: black;
border-radius: 10px; 
box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="applicationLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>760</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700  13pt &quot;Segoe UI&quot;;
text-align:center;
color: black;</string>
    </property>
    <property name="text">
     <string>     Welcome to our SE16 Weather Forecast Application!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
